systemtest/src/main/java/io/strimzi/systemtest/logs/TestExecutionWatcher.java

```diff
@@ -20,6 +20,8 @@
 
 public class TestExecutionWatcher implements TestExecutionExceptionHandler, LifecycleMethodExecutionExceptionHandler {
 
+    private static final Random RANDOM = new Random();
```

Comment:
Why do you use once static field and once non-static? Shouldn't it use everywhere the same?